src/utils.js

```diff
@@ -17,7 +17,7 @@ export const lazyloadDirective = {
     const options = {
       root: null,
       rootMargin: '0px',
-      threshold: 0.1
+      threshold: 1
```

Comment:
It's not a good way to solve the issue, what if the image takes more that 1 sec to load?
I already send you a hint for proper solution. Look closer to styles and the way - how images renders.